app/classes/query/modules/high_level_queries.rb

```diff
@@ -112,7 +112,7 @@ def paginate_ids(paginator)
         ids = ids.select { |id| @letters[id] == letter }
       end
     end
-    paginator.num_total = ids.count
+    paginator.num_total = ids.length
     ids[paginator.from..paginator.to] || []
   end
```

Comment:
@pellaea is this OK in `Query::HighLevelQueries`? At the point of figuring out the `paginator` and the `@num_results`, the query is done, so it seems `length` is more appropriate. 

It may make no difference currently, but it will probably matter if Query starts using scopes.

Reply:
Ok i now think `size` may be more appropriate.

Reply:
Oh dear god!  I can't possibly read that entire email thread right now!

I do want to make sure one thing is pointed out in this one case, though -- `ids` here is not a query, it's just a plain old `Array`.  So there's no question of it executing SQL queries or anything if "done wrong".

(Is there in fact any difference between size, count and length on a plain old Array??... oh hold on, strike that question... it's almost certainly answered somewhere in the email thread I just deleted! :)

Reply:
LOL